lib/Simulator/Simulator.cpp

```diff
@@ -83,18 +112,15 @@ void SimulatorClass::Main()
       //Pending record doesn't contain valid data, this means we've reched the end of our queue
       return;
    }
+   long lRaceElapsedTime = GET_MICROS - RaceHandler._lRaceStartTime;
    //Simulate sensors
-   if (RaceHandler.RaceState != RaceHandler.STOPPED
-      && PendingRecord.lTriggerTime <= (long) RaceHandler._lRaceTime)
+   if (RaceHandler.RaceState != RaceHandler.STOPPED && PendingRecord.lTriggerTime <= (long)lRaceElapsedTime)
    {
       if (RaceHandler._QueueEmpty())
       {
-         if ((PendingRecord.lTriggerTime < 0
-            && RaceHandler.RaceState == RaceHandler.STARTING)
-            || (PendingRecord.lTriggerTime > 0
-            && RaceHandler.RaceState == RaceHandler.RUNNING))
+         if ((PendingRecord.lTriggerTime < 0 && RaceHandler.RaceState == RaceHandler.STARTING) || (PendingRecord.lTriggerTime > 0 && RaceHandler.RaceState == RaceHandler.RUNNING))
```

Comment:
Can you please change this back to multiple lines? If all the comparison statements are on the same line, it is not possible to see everything in 1 screen when viewing the code in an editor.

Reply:
That is kind of fault in "prettier" extension of VSCode. While typically it wrap lines wider that 80 character for better visibility somehow in "if" it do exactly opposite :-(
I changed to 2 lines that I think is good solution from visibility p.o.v.. Please check and let me know. I can revert to original 4 lines if you do prefer that.